wordpress-mobile/WordPress-iOS
# Menus: Clean up white space formatting.

While developing Menus a while back, I did not have Xcode enabled for trimming whitespace.... Sooo now I've trimmed the whitespace. 😅 

Also, not entirely sure why our linter didn't catch this on its own....

To test:
- Just needs a build to ensure no compile issues.

Needs review: @aerych can I bother you for a cleanup PR? 🙏
